toml.md

```diff
@@ -36,25 +36,28 @@ should be easy to parse into data structures in a wide variety of languages.
 
 ## Spec
 
+A TOML file must be a valid UTF-8 encoded Unicode document. Specifically this
+means that, should a file as a whole not form a well-formed code-unit sequence,
```

Comment:
Why not add a link to https://unicode.org/glossary/#well_formed_code_unit_sequence here? We also link to "Unicode scalar value" elsewhere in the document, and considering that this is a fairly technical term, it would really be helpful.

Reply:
Good idea.

Reply:
On second thought, the link to the Unicode glossary could use a suitable replacement. A casual user might follow the link, get run around on the different definitions that they would need to understand, and give up. I'll use that link. But something that directly spells out what constitutes a valid document in Unicode and in UTF-8 in particular would be more helpful than that technical definition.

Reply:
Such a link, that points directly to the pdf as a deeplink to the correct section, is above in my comments. Maybe use that?

Reply:
This one: https://www.unicode.org/versions/Unicode15.0.0/ch03.pdf#G7404

Reply:
I'm sorry @abelbraaksma, but jumping into a huge PDF document would make a casual user quit all the faster. The glossary is laconic at least. So we'll stick with the link to that, and anyone who wants to dive into that rabbit hole can do so and find the Unicode specs they need.

Modern text editors know what a valid UTF-8 document should look like, after all.

Reply:
I merely responded to:

>  But something that directly spells out what constitutes a valid document in Unicode and in UTF-8 in particular would be more helpful than that technical definition.

Be careful what you wish for! (though, tbh, a link from spec to spec shouldn’t scare somebody who’s capable of interpreting ABNF. Then again, 99 out of 100 will just use a library that does the UTF8 reading, which gives all this for free)

Reply:
> I merely responded to:
> 
> >  But something that directly spells out what constitutes a valid document in Unicode and in UTF-8 in particular would be more helpful than that technical definition.
> 
> Be careful what you wish for!

In retrospect, I was asking for two diametrically opposed texts. One that was short, sweet, directed at casual readers, and already well accepted by UTF-8 encoder libraries; that's what I actually wanted. The other was the entirety of the Unicode spec detailing UTF-8. Which you directed me to.

@abelbraaksma Sorry for the confusion. Part of me wants to keep the spec as direct as possible. But those days are gone. Frankly, [toml.io](https://toml.io)'s home page is as direct as we'll ever get.